packages/mdc-button/README.md

```diff
@@ -126,7 +126,7 @@ Buttons are fully aware of ripple styles, so no DOM or CSS changes are required
 
 ### Sass Mixins
 
-By default an MDC Button will inherit its color from the theme and align with [Material Design button requirements](https://material.io/guidelines/components/buttons.html). To customize a Button's color and properties, you can use the following mixins.
+By default an MDC Button will inherit its color from the theme and align with [Material Design button requirements](https://material.io/go/design-buttons). To customize a Button's color and properties, you can use the following mixins.
```

Comment:
FYI I'm pretty sure this was also changed in the recently-merged PR and the link no longer exists.

Reply:
Updated.